rpc/src/client/transport/websocket.rs

```diff
@@ -586,7 +586,7 @@ mod test {
             loop {
                 tokio::select! {
                     Some(ev) = self.event_rx.recv() => self.publish_event(ev),
-                    Some(res) = self.listener.next() => self.handle_incoming(res.unwrap()).await,
+                    Ok((stream, _)) = self.listener.accept() => self.handle_incoming(stream).await,
```

Comment:
@thanethomson This is imho cleaner as it avoids an `unwrap` but will silently discard errors when accepting a new connection whereas the previous solution would panic. Does this work for you?

Reply:
In testing code I'd generally prefer `.unwrap()` (personal preference). Absolutely nothing should go wrong here, and the test should break clearly and definitively if something fails.

Reply:
Makes sense! Fixed in f8a37a67034122dd8e11e522716b10b33c219b34.